### DISCUSSION TITLE
Backup di 8.9 GB.

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#1335] [HUB: Home Assistant]
Buongiorno, stamattina ho trovato la sorpresa di un backup di 9 gb, normalmente è di 3 gb. Uso l'addon 'backups' per la sincronizzazione su drive, però manca lo spazio e non riesce a fare il trasferimento, qualcuno sa dirmi dove guardare per questa anomalia?
Dimenticavo: faccio la pulizia disco tutti i giorni di default con purge, il DB SQlite si mantiene di 700/800 Kb.

**Comment by snakuzzo** [comment]
Sì ma con questo che hai scritto nessuno può aiutarti.
Cosa fa questo backup ? Cosa c'è dentro il backup ? Hai controllato quali file occupano tutto questo spazio ?

**Reply by calz64** [reply]
E' il backup full dell'addon  Backups (vedi foto, l'ultimo è di 8,9 GB, in un anno che fa backup non aveva mai fatto un  bk cosi grande) , francamente credo ci sia tutto il sistema perché ho reinstallato un paio di volte con quello e in effetti riapri con tutto. :)
![Cattura](https://user-images.githubusercontent.com/84615011/149829386-33762b07-ee4b-4e3f-8d47-250337442f78.JPG)

**Reply by snakuzzo** [reply]
Ma che ci fai con tutta sta roba?

**Reply by calz64** [reply]
E' un servizio di supevisor; si possono impostare i bk che si vogliono, forse tre sono troppi, ma in ogni caso l'anomalia è i 9 GB non capisco perchè questo giro è 9 Gb, non c'è modo di capirlo?

**Reply by snakuzzo** [reply]
Prendi il backup, scaricalo, e vedi che c'è dentro

**Reply by calz64** [reply]
Seguendo la guida dall'inizio ho fatto i bk con questo addon 'backups' con sincro in Drive (comodo), però al momento la cartella /share sta aumentando per le registrazioni delle telecamere di sicurezza, torno al bk di Supervisor, che permette di selezionare i file/cartelle da sincronizzare, purtroppo ad oggi 'backups' non lo permette, almeno io ho capito così.
